src/ShareByEmail.java

```diff
@@ -0,0 +1,12 @@
+public class ShareByEmail implements Shareable {
```

Comment:
And here the small explenation of the comment 
What I really don’t like is that there is a Ticket here.
I don’t quite understand the point of this. If I want to send a different kind of ticket, do I need to create another SendByEmail class?

Wouldn’t it be better to create a structure like this:

You will have smth like 

`interface Shareable` and lets make it empty for now

than `Ticket` will implement `Shareable`
 After it you will have some new interface 

`interface Sander` naming up to you,
with method void send(share : Shareable)


that the call will be like this
shareByMobile.send(ticket)

Moreover you will have an option to send other staff
What if you will need to send client data or user, you will just add that interface to them , but in your case you will add new class which will be able just tu send clien or smth else